cluster-up/cluster/kind/podpreset.sh

```diff
@@ -37,6 +38,11 @@ EOT
     fi
 }
 
+function podpreset::wait_for_kube_system_components_ready() {
+    local -r timeout=$1
```

Comment:
What does the `-r` flag do?

Reply:
Its an option of `local`, it sets the variable to be readonly

Reply:
Thanks!